src/EasyCaching.Core/Decoration/ICircuitBreakerParameters.cs

```diff
@@ -0,0 +1,11 @@
+namespace EasyCaching.Core.Decoration
+{
+    using Polly;
+
+    public interface ICircuitBreakerParameters
+    {
+        Policy<TResult> CreatePolicy<TResult>(PolicyBuilder<TResult> policyBuilder);
+        Policy CreatePolicy(PolicyBuilder policyBuilder);
+        AsyncPolicy CreatePolicyAsync(PolicyBuilder policyBuilder);
```

Comment:
`Polly` should not be directly depend on here. We should make an abstraction, `Polly` is one of the implementation.

Reply:
If we find another library for circuit breakers it may have setting incompatible with polly settings. So I moved CBParameters to EasyCaching.Decoration.Polly for now. 
If we find another library with compatible settings, then we may do a common abstraction for CBParameters in Core.